src/HackF5.UnitySpy/Detail/AssemblyImage.cs

```diff
@@ -81,16 +81,16 @@ public TypeDefinition GetTypeDefinition(IntPtr address)
         {
             var definitions = new ConcurrentDictionary<IntPtr, TypeDefinition>();
             int classCache = this.Process.MonoLibraryOffsets.ImageClassCache;
-            var classCacheSize = this.ReadUInt32(classCache + this.Process.MonoLibraryOffsets.HashTableSize);
-            var classCacheTableArray = this.ReadPtr(classCache + this.Process.MonoLibraryOffsets.HashTableTable);
+            var classCacheSize = this.ReadUInt32(/*classCache + this.Process.MonoLibraryOffsets.HashTableSize*/1256);
```

Comment:
Here I only know the good values for `classeCache + HashTableSize` and `classCache + HashTableTable` but I have no idea about the individual values of each offset